src/zosmf_workflows/setup.py

```diff
@@ -0,0 +1,42 @@
+"""Zowe Client Python SDK.
+
+This program and the accompanying materials are made available under the terms of the
+Eclipse Public License v2.0 which accompanies this distribution, and is available at
+
+https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zowe Project.
+
+Contributors:
+    Zowe Community
+    Uladzislau Kalesnikau
```

Comment:
I haven't seen this before from Zowe core contributors, but that's cool! 😅 

Is this something we want to start doing going forward? 🤔 

If so, we may want to adjust some license_header scripts on other repos.

Personally, I'd vote for sticking to the Git sign-offs instead of adding individuals names to each file since it can get out of hand 😅 
 
If someone is concerned about the legitimacy of git sign-offs, there is the possibility to configure GPG signing for each commit made to the repo! 🙏